### ISSUE TITLE
Split up lib/src/meta.dart into different files

### ISSUE THREAD

**Issue by lukas-h** [dartclub/turf_dart#60]
The implementation file for the meta functions library gets really long and we decided to split it up into a file for each function.

Like this:
`lib/src/meta/geom_each.dart`  
`lib/src/meta/coord_each.dart`

But before implementing this, I would suggest waiting for the merging of the meta functions that are already in development (see status: #12), so it doen't get messy!

**Comment by lukas-h** [comment]
We should do that, after #70 and #69 are merged @armantorkzaban
